data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_pre_2002.Rmd

```diff
@@ -0,0 +1,770 @@
+  select(week, number_marked, number_recovered) %>% 
+  filter(week != "Total") %>% 
+  mutate(year = 2001,
+         week = as.numeric(week)) %>% 
+  rename(number_recaptured = number_recovered)
+```
+
+# Combine 1996-2001
+
+trap: date, time, trapid, effort, avg_rpm, velocity
+rst: date, trapid, fl, ww, lifestage, race, mort, clip, mark, id_num
+efficiency: date, week, year, number_marked, number_recaptured
+marked: date_released, time_released, number_released
+recaptured: date, time, trapid, fl, ww, lifestage, race
+
+## trap
+```{r}
+trap_combined <- bind_rows(trap96_clean,
+                           trap97_clean,
+                           trap98_clean,
+                           trap99_clean,
+                           trap00_clean,
+                           trap01_clean) %>% glimpse()
+```
+### time
+```{r}
+trap_combined %>% group_by(year = year(date), time) %>% tally()
+```
+
+### trapid
+
+We will need to figure out how to assign these trapids to the current 8.3 and 
+8.4 sites for Knights Landing
+
+```{r}
+unique(trap_combined$trapid)
+trap_combined %>% group_by(year = year(date), trapid) %>% tally()
+```
+
+### effort
+```{r}
+trap_combined %>% 
+  ggplot(aes(x = as.factor(year(date)), y = effort)) +
+  geom_boxplot()
+```
+
+### avg_rpm
+```{r}
+trap_combined %>% 
+  ggplot(aes(x = as.factor(year(date)), y = avg_rpm)) +
+  geom_boxplot()
+```
+
+### velocity
+```{r}
+trap_combined %>% 
+  ggplot(aes(x = as.factor(year(date)), y = velocity)) +
+  geom_boxplot()
+```
+## rst 
+
+Codes for Life Stage from Jeanine Phillips:
+During the early years, the codes were explained on the datasheets, 
+but they are the ones still in use today: 1=alevin, 2=fry, 3=parr, 4=silvery parr, 5=smolt
+
+Codes for Run from Jeanine Phillips:
+1=fall, 2=spring, 3=winter, 4=late fall
+Assume 5 is unknown
+
+```{r}
+rst_combined <- bind_rows(rst96_clean,
+                          rst97_clean,
+                          rst98_clean,
+                          rst99_clean,
+                          rst00_clean,
+                          rst01_clean) %>% 
+  mutate(lifestage = case_when(lifestage == 1 ~ "alevin",
+                               lifestage == 2 ~ "fry",
+                               lifestage == 3 ~ "parr",
+                               lifestage == 4 ~ "silvery parr",
+                               lifestage == 5 ~ "smolt"),
+         run = case_when(race == 1 ~ "fall",
+                         race == 2 ~ "spring",
+                         race == 3 ~ "winter",
+                         race == 4 ~ "late fall",
+                         race == 5 ~ "unknown")) %>% 
+  select(-race) %>% 
+  glimpse()
+```
+### time
+```{r}
+rst_combined %>% group_by(year = year(date), time) %>% tally()
+```
+
+### trapid
+```{r}
+rst_combined %>% group_by(year = year(date), trapid) %>% tally()
+```
+
+### fl
+```{r}
+rst_combined %>% 
+  ggplot(aes(x = as.factor(year(date)), y = fl)) +
+  geom_boxplot()
+
+rst_combined %>% 
```

Comment:
looks like there is one negative fl. We may want to remove this one

Reply:
good catch. i assumed that instead of -83 they meant 83 and fixed this.